types.bal

```diff
@@ -0,0 +1,672 @@
+#               whether the current user can copy non-folder 
+#               descendants of this item, or this item itself if it is not a folder.
+# + canDownload -   Whether the current user can download this file.
+# + canDelete -  Whether the current user can delete this file. 
+# + canAddFolderFromAnotherDrive -  Whether the current user can add a folder from 
+#                                   another drive (different shared drive or My Drive) to this folder. 
+#                                   This is false when the item is not a folder. Only populated for items 
+#                                   in shared drives. 
+# + canComment - Whether the current user can comment on this file.  
+# + canUntrash - Whether the current user can restore this file from trash.   
+# + canMoveChildrenWithinDrive -   Whether the current user can move children of this folder within this drive. 
+#                                  This is false when the item is not a folder. 
+#                                  Note that a request to move the child may still fail depending on the 
+#                                  current user's access to the child and to the destination folder
+# + canModifyContentRestriction -   Whether the current user can modify restrictions on content of this file  
+# + canChangeCopyRequiresWriterPermission -   Whether the current user can change the copyRequiresWriterPermission 
+#                                             restriction of this file.
+# + canMoveChildrenOutOfDrive -  Whether the current user can move children of this folder outside of the shared drive. 
+#                                This is false when the item is not a folder.Only populated for items in shared drives. 
+# + canReadDrive -  Whether the current user can read the shared drive to which this file belongs. 
+#                   Only populated for items in shared drives. 
+# + canShare -   Whether the current user can modify the sharing settings for this file.
+# + canDeleteChildren -   Whether the current user can delete children of this folder. 
+#                         This is false when the item is not a folder. 
+#                         Only populated for items in shared drives.
+# + canMoveItemWithinDrive -   Whether the current user can move this item within this drive. 
+#                              Note that a request to change the parent of the item may still fail 
+#                              depending on the new parent that is being added and the parent that is being removed. 
+# + canModifyContent -  	Whether the current user can modify the content of this file. 
+# + canRemoveChildren -   Whether the current user can remove children from this folder. 
+#                         This is always false when the item is not a folder. 
+#                         For a folder in a shared drive, use canDeleteChildren or canTrashChildren instead.   
+public type Capabilities record {
+    boolean	canAddChildren;
```

Comment:
formatting issue

Reply:
I think you are referring to  record type documentation right?
Made some changes on the comments. Let me know whether I did it correctly @LakshanSS

Reply:
I meant one space after the boolean

Reply:
Thanks. I updated the code.